### ISSUE TITLE
Allow for Packaging System to also resolve .NET version dependencies

### ISSUE THREAD

**Issue by bdirenzo-keysight** [opentap/opentap#486]
With .NET 6, so plugins will still require .NET Framework. Is it possible to resolve and automatically install .NET versions as part of the plugin install? Maybe this only requires a .NET Framework plugin in the repo. There may also be better ways to address this.

**Comment by bdirenzo-keysight** [comment]
@alnlarsen not sure if you have any thoughts on this. I know have been looking at some of the dependency things.

**Comment by rmadsen-ks** [comment]
Maybe .net 6 could be installed with a "Virtual Package" #493

**Comment by bdirenzo-keysight** [comment]
Yes, seems that could address this. I've added a comment over there.
